lib/routes/hiringcafe/jobs.ts

```diff
@@ -0,0 +1,102 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+
+export const route: Route = {
+    path: '/jobs/:keywords',
+    categories: ['other'],
+    example: '/hiringcafe/jobs/sustainability/',
+    parameters: { keywords: 'Keywords to search for' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['hiring.cafe'],
+        },
+    ],
+    name: 'HiringCafe Jobs',
```

Comment:
No need to include namespace name
https://github.com/DIYgod/RSSHub/blob/12a30ed0c26efca497ac1e24a630319a14614c26/lib/routes/hiringcafe/namespace.ts#L4
in route name again